app/api/answers/generate_task_token.feature

```diff
@@ -0,0 +1,251 @@
+Feature: Generate a read-only task token for an item from an answer
+  Background:
+    Given the database has the following table 'groups':
+      | id  | name     | type  |
+      | 101 | john     | User  |
+      | 102 | team     | Team  |
+      | 103 | manager  | User  |
+      | 104 | Groupe A | Class |
+    And the database has the following table 'users':
+      | login | group_id |
+      | john    | 101    |
+      | manager | 103    |
+    And the database has the following table 'groups_groups':
+      | parent_group_id | child_group_id |
+      | 102             | 101            |
+      | 104             | 101            |
+    And the database has the following table 'group_managers':
+      | group_id | manager_id | can_watch_members |
+      | 104      | 103        | 1                 |
+    And the groups ancestors are computed
+    And the database has the following table 'items':
+      | id | url                                                                     | type | entry_participant_type | default_language_tag | text_id |
+      | 10 | http://taskplatform.mblockelet.info/task.html?taskId=403449543672183936 | Task | User                   | fr                   | task10  |
+      | 20 | http://taskplatform.mblockelet.info/task.html?taskId=403449543672183936 | Task | Team                   | fr                   | task20  |
+      | 30 | http://taskplatform.mblockelet.info/task.html?taskId=403449543672183936 | Task | User                   | fr                   | task30  |
+      | 40 | http://taskplatform.mblockelet.info/task.html?taskId=403449543672183936 | Task | User                   | fr                   | task40  |
+      | 50 | http://taskplatform.mblockelet.info/task.html?taskId=403449543672183936 | Task | User                   | fr                   | task50  |
```

Comment:
if all urls are the same, we cannot check they are correct. BTW, use `http://taskplatform/10`, that's easier to read :-)
(beware that the `url` may be related to the `platform` table which provides the keys for signing token ??? to be checked)

Reply:
fixed